core/object/script_language.h

```diff
@@ -304,7 +310,8 @@ class ScriptLanguage : public Object {
 	virtual int find_function(const String &p_function, const String &p_code) const = 0;
 	virtual String make_function(const String &p_class, const String &p_name, const PackedStringArray &p_args) const = 0;
 	virtual Error open_in_external_editor(const Ref<Script> &p_script, int p_line, int p_col) { return ERR_UNAVAILABLE; }
-	virtual bool overrides_external_editor() { return false; }
+	virtual bool overrides_external_editor() const { return false; }
```

Comment:
AFAIK such changes break compatibility with GDExtension.

Reply:
I will revert the change, since it is not required for this PR. Not worth adding the overhead, if it's not even needed here and now.